src/test/java/io/strimzi/kafka/bridge/http/ConsumerTest.java

```diff
@@ -208,11 +211,12 @@ void createConsumerWithForwardedHeaders(VertxTestContext context) throws Interru
                     context.completeNow();
                 });
 
+        consumerService().deleteConsumer(context, groupId, name);
```

Comment:
Two things:

* I don't think you can do in this way when `context.completeNow' is called async and could end the test before deleting. You need a `CompletableFuture` as done in other tests.
* delete on the next line :-)